src/type/Popover.ts

```diff
@@ -0,0 +1,6 @@
+export interface popoverProps {
+  position?: 'top' | 'right' | 'bottom' | 'left'
+  heading?: string
+  icon?: string | object
+  contentItems?: string[]
```

Comment:
If we didn't give an ID to each item, then how can we find which item we have selected or not?